UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -18,6 +18,15 @@ namespace Chemistry.Components
 	public partial class ReagentContainer : MonoBehaviour, IServerSpawn, IRightClickable, ICheckedInteractable<ContextMenuApply>,
 		IEnumerable<KeyValuePair<Reagent, float>>
 	{
+		[Flags]
+		private enum ShowMenuOptions
+		{
+			None = 0,
+			ShowContents = 1 << 0,
+			PourOut = 1 << 1,
```

Comment:
true, though having it set up automatically is a bonus, 

do you think it would be to be complicated if it used the bool and the enum would override it?

Reply:
It properly should do, I think having the Enumerator would be best option if we can't have both

Reply:
I'll see about removing the old bool tonight and adjusting the prefabs that had changed it. If there ends up being a significant number of prefab changes caused by the removal, then I'll leave it alone for this PR.